ast/spec.md

```diff
@@ -149,6 +151,15 @@ interface Identifier <: Expression, Pattern {
 }
 ```
 
+# PrivateName
+
+```js
+interface PrivateName <: Expression, Pattern {
```

Comment:
#### Identifier

It is an identifier with a specific type.
It would be confusing for developers because they will be forced to check the type of the identifier.

Most of them will omnit it and will cause some bugs.


#### PrivateName

More explicit.

I don't have much context here, but I guess most transformation within a class will be with both PrivateNames and Identifier.